### ISSUE TITLE
Failure when building from AUR

### ISSUE THREAD

**Issue by godofgrunts** [M-Reimer/wine-lol-glibc#3]
Installing with `yay -S wine-lol-glibc`, I fail with the following output:

```
make[1]: Leaving directory '/home/godofgrunts/.cache/yay/wine-lol/src/wine-lol-32-build/libs/port'
make[1]: Entering directory '/home/godofgrunts/.cache/yay/wine-lol/src/wine-lol-32-build/tools'
LC_ALL=C sed -e 's,@bindir\@,/opt/wine-lol/bin,g' -e 's,@dlldir\@,/opt/wine-lol/lib32/wine,g' -e 's,@srcdir\@,../../wine-lol/tools,g' -e 's,@PACKAGE_STRING\@,Wine 4.8,g' -e 's,@PACKAGE_VERSION\@,4.8,g' ../../wine-lol/tools/wineapploader.in >wineapploader || (rm -f wineapploader && false)
gcc -m32 -c -o make_xftmpl.o ../../wine-lol/tools/make_xftmpl.c -I. -I../../wine-lol/tools -I../include \
  -I../../wine-lol/include -D__WINESRC__ -Wall -pipe -fno-strict-aliasing \
  -Wdeclaration-after-statement -Wempty-body -Wignored-qualifiers -Wno-packed-not-aligned \
  -Wshift-overflow=2 -Wstrict-prototypes -Wtype-limits -Wunused-but-set-parameter -Wvla \
  -Wwrite-strings -Wpointer-arith -Wlogical-op -fno-omit-frame-pointer -D_FORTIFY_SOURCE=2 -m32 -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=0
gcc -m32 -o make_xftmpl make_xftmpl.o ../libs/port/libwine_port.a -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now,-rpath=/opt/wine-lol/lib32,-dynamic-linker=/opt/wine-lol/lib32/ld-linux.so.2
make[1]: Leaving directory '/home/godofgrunts/.cache/yay/wine-lol/src/wine-lol-32-build/tools'
make[1]: Entering directory '/home/godofgrunts/.cache/yay/wine-lol/src/wine-lol-32-build/tools/sfnt2fon'
gcc -m32 -c -o sfnt2fon.o ../../../wine-lol/tools/sfnt2fon/sfnt2fon.c -I. -I../../../wine-lol/tools/sfnt2fon \
  -I../../include -I../../../wine-lol/include -I/usr/include/freetype2 -I/usr/include/libpng16 \
  -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib32/glib-2.0/include -D__WINESRC__ -Wall \
  -pipe -fno-strict-aliasing -Wdeclaration-after-statement -Wempty-body -Wignored-qualifiers \
  -Wno-packed-not-aligned -Wshift-overflow=2 -Wstrict-prototypes -Wtype-limits \
  -Wunused-but-set-parameter -Wvla -Wwrite-strings -Wpointer-arith -Wlogical-op \
  -fno-omit-frame-pointer -D_FORTIFY_SOURCE=2 -m32 -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=0
gcc -m32 -o sfnt2fon sfnt2fon.o ../../libs/port/libwine_port.a -L/usr/lib32 -lfreetype -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now,-rpath=/opt/wine-lol/lib32,-dynamic-linker=/opt/wine-lol/lib32/ld-linux.so.2
/usr/bin/ld: ../../libs/port/libwine_port.a(mbtowc.o): direct GOT relocation R_386_GOT32X against `__stack_chk_fail@@GLIBC_2.4' without base register can not be used when making a shared object
/usr/bin/ld: final link failed: nonrepresentable section on output
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:190: sfnt2fon] Error 1
make[1]: Leaving directory '/home/godofgrunts/.cache/yay/wine-lol/src/wine-lol-32-build/tools/sfnt2fon'
make: *** [Makefile:8945: tools/sfnt2fon] Error 2

```

Here is my `gcc -v`

```
Using built-in specs.
COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-pc-linux-gnu/8.3.0/lto-wrapper
Target: x86_64-pc-linux-gnu
Configured with: /build/gcc/src/gcc/configure --prefix=/usr --libdir=/usr/lib --libexecdir=/usr/lib --mandir=/usr/share/man --infodir=/usr/share/info --with-bugurl=https://bugs.archlinux.org/ --enable-languages=c,c++,ada,fortran,go,lto,objc,obj-c++ --enable-shared --enable-threads=posix --enable-libmpx --with-system-zlib --with-isl --enable-__cxa_atexit --disable-libunwind-exceptions --enable-clocale=gnu --disable-libstdcxx-pch --disable-libssp --enable-gnu-unique-object --enable-linker-build-id --enable-lto --enable-plugin --enable-install-libiberty --with-linker-hash-style=gnu --enable-gnu-indirect-function --enable-multilib --disable-werror --enable-checking=release --enable-default-pie --enable-default-ssp --enable-cet=auto
Thread model: posix
gcc version 8.3.0 (GCC) 
```

Here's my `neofetch --off`

```neofetch --off
godofgrunts@itx-dev.local 
------------------------- 
OS: Arch Linux x86_64 
Kernel: 5.1.4-arch1-1-ARCH 
Uptime: 1 hour, 36 mins 
Packages: 1296 (pacman) 
Shell: zsh 5.7.1 
Resolution: 2560x1080, 1920x1080, 1920x1200 
DE: Cinnamon 4.0.10 
WM: Mutter (Muffin) 
WM Theme: cinnamon (Adwaita) 
Theme: Adwaita [GTK2/3] 
Icons: Adwaita [GTK2/3] 
Terminal: terminator 
CPU: AMD Ryzen 5 2400G (8) @ 3.600GHz 
GPU: AMD ATI Radeon RX 470/480/570/570X/580/580X/590 
Memory: 2765MiB / 16033MiB 
```

**Comment by godofgrunts** [comment]
Strange. Builds fine if I just download the PKGBUILD. Must be an issue with `yay`

I'll let you confirm this though and close if you'd like.

**Comment by M-Reimer** [comment]
This is known.
I have no idea why the wine-lol PKGBUILD works without yay but fails if you try to build with this helper.
I'll open a bug report for yay. Maybe the developer tries this out and can tell why this doesn't work.

**Comment by godofgrunts** [comment]
>This is known.

Thanks for confirming, you should put that in the AUR comments or a README file until the issue is resolved so others don't come bothering you.

**Comment by M-Reimer** [comment]
I've added a possible "yay fix" to the 4.9 PKGBUILD.
Maybe you could try to build this with yay so I can close the upstream bug?

**Comment by godofgrunts** [comment]
Just removed both packages and installed again with `yay -S wine-lol` which installed `wine-lol-glibc` and `wine-lol` perfectly.

Thanks for looking into this!
